### PR TITLE
Adding upgrade to compaign smoke

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | New smoke test upgradeShop
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="smoke/upgradeShop" URL_FO=SHOP_URLFO npm rum specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/14920)

### CODE REVIEW

**Comment by boubkerbribri** [comment]
WIP : Need PR #14902 to add checks in FO

File: tests/puppeteer/test/linkchecker.js [hunk 1]
```diff
@@ -127,7 +127,8 @@ const run = async () => {
 
   const curDate = new Date();
 
-  const filename = `report_${curDate.toJSON().slice(0, 10)}_${curDate.getHours()}${curDate.getMinutes()}${curDate.getSeconds()}.json`;
+  const filename = `report_${curDate.toJSON()
+    .slice(0, 10)}_${curDate.getHours()}${curDate.getMinutes()}${curDate.getSeconds()}.json`;
```
Comment (SimonGrn) [inline tests/puppeteer/test/linkchecker.js]: Linter made this ?

Comment (boubkerbribri) [reply tests/puppeteer/test/linkchecker.js]: yes,
I add "max-len": [2, {"code": 120}] in linter and the error appear . so going to new line makes error disappear.

Comment (PierreRambaud) [reply tests/puppeteer/test/linkchecker.js]: better to present like this?
```js
const dateString = curDate.toJSON().slice(0, 10);
const hours = curDate.getHours();
const minutes = curDate.getMinutes();
const seconds = curDate.getSEconds();
const filename = `report_${dateString}_${hours}${minutes}${seconds}.json`;
```

Comment (boubkerbribri) [reply tests/puppeteer/test/linkchecker.js]: done thanks

**Comment by Quetzacoalt91** [comment]
Testing the module interface should be done in the module repository. For your needs, you could run an upgrade in command line (quick & easy) from a given version of PrestaShop.

**Comment by SimonGrn** [comment]
The t

> Testing the module interface should be done in the module repository. For your needs, you could run an upgrade in command line (quick & easy) from a given version of PrestaShop.

Thing is, we need to test the most common upgrade path.

**Pull request closed without merging**